applications/ParachainPulse.md

```diff
@@ -0,0 +1,219 @@
+| 0d. | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains what was achieved as part of the grant and how users can benefit the most from ParachainPulse functionalities. |
+| 1. | Implement Github scraper functionality | Periodically update the list of repositories that are being part of the Polkadot & Kusama and each of their parachains on GitHub |
+| 2. | Implement Github scraper functionality | Calculate the total number of commits, repositories and contributors within the Polkadot & Kusama and each of their parachains |
+| 3. | Implement Github scraper functionality | Get the contributors of the month based on the number of commits for Polkadot & Kusama and each of their parachains |
+| 4. | Implement Github scraper functionality | Calculate the total number of commits across all the repositories for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 5. | Implement Github scraper functionality | Calculate the number of active repositories for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 6. | Implement Github scraper functionality | Calculate the number of active contributors for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 7. | Implement Github scraper functionality | Collect the list of recent commits across all the Polkadot's Github repositories from the last 30 days |
+| 8. | Implement the ParachainPulse APIs required by frontend | Statistics API that returns the number of monthly active projects, monthly active developers and monthly contributions for Polkadot & Kusama |
+| 9. | Implement the ParachainPulse APIs required by frontend | Growth API that returns the active monthly growth for developers, projects and contributions over the last 12 months for Polkadot & Kusama and each of their parachains |
+| 10. | Implement the ParachainPulse APIs required by frontend | Developers API that returns the list of contributors, the number of contributors and active contributors for Polkadot & Kusama and each of their parachains |
+| 11. | Implement the ParachainPulse APIs required by frontend | Projects APIs that return the list of projects, the total number projects, the active projects during the last 30 days and during the last 12 months for Polkadot & Kusama and each of their parachains |
+| 12. | Implement the ParachainPulse APIs required by frontend | Contributions API that returns the total number of commits, the number of commits during the last 30 days, and during the last 6 months and commits growth for 12 months for Polkadot & Kusama and each of their parachains |
+| 13. | Implement the ParachainPulse APIs required by frontend | Submit parachain API |
```

Comment:
Can you elaborate on _13. Submit parachain API_? A brief description will help us to better evaluate this part.

Reply:
“Submit Parachain” API is used by the frontend to push the following data to the the database: ‘Parachain name’, ‘GitHub URL’, ‘Logo URL’, ‘Email’. The admin reviews it and, if approved, is added to the parachain list. Additionally, the scraper will retrieve relevant data about the approved parachain and fill in the details automatically.